src/game-engine/state/reducers/game.reducer.js

```diff
@@ -184,6 +184,20 @@ export function game(state = initialState, action) {
 				cardSelected: action.data
 			}
 		}
+		case gameConstants.RECONNECT_SUCCESS:
+			return {
+				...state,
+				playerData: {
+					id: action.data.player._id,
+					position: action.data.player.position,
+					name: action.data.player.name,
+					hand: action.data.player.hand
+				},
+				gameData: {
```

Comment:
Is there any reason you aren't adding the logs and other details to store here ?

Reply:
Yeah damn, didn't notice this. Why can't you just store the whole data that you get?